vertexai/snippets/try_gemini.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snippets
+
+// [START aiplatform_gemini_get_started]
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/vertexai/genai"
+)
+
+var projectId = "PROJECT_ID"
+var region = "us-central1"
+
+func tryGemini(w io.Writer, projectId string, region string, modelName string) error {
+
+	client, err := genai.NewClient(context.TODO(), projectId, region)
+	gemini := client.GenerativeModel("gemini-pro-vision")
+
+	img := genai.FileData{
+		MIMEType: "image/jpeg",
+		FileURI:  "https://storage.googleapis.com/generativeai-downloads/images/scones.jpg",
```

Comment:
Interesting: this uses `FileURI` directly, while Hussain's samples implement reading/loading from a path?

Consistency would be good here too :)

Reply:
Agreed. Hussain and I chatted about it today and agree the `FileData` approach leads to "cleaner" samples and we'll adopt it where we can.